frame/support/src/dispatch.rs

```diff
@@ -2169,21 +2175,27 @@ mod tests {
 	pub struct TraitImpl {}
 
 	impl Trait for TraitImpl {
-		type Origin = u32;
 		type BlockNumber = u32;
-		type Call = OuterCall;
 	}
 
+	/// HREHREHREHRE
```

Comment:
?

Reply:
sorry removed :-/ (it was to see macro expansion)